lib/core/src/lib/datatable/components/datatable/datatable.component.ts

```diff
@@ -950,7 +950,7 @@ export class DataTableComponent implements OnInit, AfterContentInit, OnChanges,
 
     onResizing({ rectangle: { width } }: ResizeEvent, colIndex: number): void {
         const timeoutId = setTimeout(() => {
```

Comment:
Not sure why timeout is cleared this way. Is it cleared to avoid memory leaks? If yes are you sure that clearing timeout in this way works? I think it will not work. I would rather create array with all timeout id's and clear them in onDestroy.